.env

```diff
@@ -1,5 +1,5 @@
 OPENID_GIT_URL=https://gitlab.com/openid/conformance-suite.git
-OPENID_GIT_TAG=${OPENID_GIT_TAG:-release-v4.1.38}
+OPENID_GIT_TAG=${OPENID_GIT_TAG:-release-v4.1.39}
```

Comment:
Could you please test it with the latest conformance suite version ( release-v4.1.41) - https://gitlab.com/openid/conformance-suite/-/tags ?

Reply:
Thanks for the hint! Just updated the PR

Reply:
Thanks for the hint! Just updated the PR